modules/shared/components/atoms/VoteImage/VoteImage.tsx

```diff
@@ -0,0 +1,36 @@
+import type { ReactElement, FC } from 'react';
+import React from 'react';
+import type { IVoteImage } from './IVoteImage';
+import OutlinedHeartIcon from '../../icons/outlinedHeart.svg';
+import FilledHeartIcon from '../../icons/filledHeart.svg';
+import OutlinedDislikeIcon from '../../icons/outlinedDislike.svg';
+import OutlinedLikeIcon from '../../icons/outlinedLike.svg';
+import FilledLikeIcon from '../../icons/filledLike.svg';
+import styles from './VoteImage.module.css';
+
+const VoteImage: FC<IVoteImage.IProps> = ({ oneImage }): ReactElement => {
+  return (
+    <div className="relative">
+      <div>
+        {oneImage ? (
```

Comment:
The code is working fine 👍👍 but I think it will be more readable if you use  AND operator (&&)
```
{ true && <ComponentOne /> }
{ false && <ComponentTwo /> }
```